web/src/test/manual/web/unminified.html

```diff
@@ -46,6 +46,10 @@
 
     <!-- Add keyboard management script for local selection of keyboards to use -->
     <script src="commonHeader.js"></script>
+    <script src="kmpHeader.js"></script>
+
+    <!-- Add local KMP loading script -->
+    <script src="../../../../build/tools/testing/bulk_rendering/lib/zip.min.js" type="application/javascript"></script>
```

Comment:
Should we just keep another copy in web/src/test/manual/web/ ?

Reply:
Note that the linked file is in the `web/build/` folder's subdirectory structure.  It's fine being there.

Reply:
Does /web/build/ get propagated where KeymanWeb test pages get "published"?

For example:
https://downloads.keyman.com/web/alpha/18.0.175/static/src/test/manual/web/unminified.html

Reply:
Yes it does - if it didn't, the test pages wouldn't work.  Also, I just pulled up the test-page artifact; the KMP load works there.